test/README.md

```diff
@@ -8,5 +8,16 @@ The ingress-node-firewall e2e tests are divided into 2 suites:
 
 To run the tests use the following make targets:
 - make test-validation - run the validation tests
+    - Enable running e2e tests in a non-default namespace by setting environment variable `OO_INSTALL_NAMESPACE` value to the namespace name.
 - make test-functional - run the functional tests
+    - Enable running e2e tests in a non-default namespace by setting environment variable `OO_INSTALL_NAMESPACE` value to the namespace name.
+    - Override the default test interface (currently "eth0") for installing the XDP program by setting environment variable `NODE_INTERFACE` value to the interface name. Overriding is required when testing on OCP and must be set to `genev_sys_6081` when OVN-Kubernetes is the CNI.
+    - SCTP tests are enabled by default. Your test target kubernetes cluster nodes must have have SCTP kernel modules loaded.
+      Enable the kernel module on RHEL 8 / CoreOS:
+      ```shell
+      modprobe sctp
+      ```
+      Disable SCTP tests by setting environment variable `DISABLE_SCTP=true`
+    - By default, tests are for IPV4 and IPV6. Disable IPV6 by setting environment variable `IS_SINGLESTACK=true`.
```

Comment:
`IS_SINGLESTACK` doesn't disable IPv6, it just allow the e2e to run on single stack cluster whether its IPv4 or IPv6